cmd/rigs/cmd/publish.go

```diff
@@ -187,3 +195,29 @@ func getChain() (client.Chain, error) {
 		return client.Chain{}, fmt.Errorf("%s is not a valid chain", chain)
 	}
 }
+
+var infuraURLs = map[client.ChainID]string{
+	client.ChainIDs.EthereumGoerli: "https://goerli.infura.io/v3/%s",
+	client.ChainIDs.Ethereum:       "https://mainnet.infura.io/v3/%s",
+	client.ChainIDs.OptimismGoerli: "https://optimism-goerli.infura.io/v3/%s",
+	client.ChainIDs.Optimism:       "https://optimism-mainnet.infura.io/v3/%s",
+	client.ChainIDs.ArbitrumGoerli: "https://arbitrim-goerli.infura.io/v3/%s",
+	client.ChainIDs.Arbitrum:       "https://arbitrum-mainnet.infura.io/v3/%s",
+	client.ChainIDs.PolygonMumbai:  "https://polygon-mumbai.infura.io/v3/%s",
+	client.ChainIDs.Polygon:        "https://polygon-mainnet.infura.io/v3/%s",
+}
+
+var alchemyURLs = map[client.ChainID]string{
+	client.ChainIDs.EthereumGoerli: "https://eth-goerli.g.alchemy.com/v2/%s",
+	client.ChainIDs.Ethereum:       "https://eth-mainnet.g.alchemy.com/v2/%s",
+	client.ChainIDs.OptimismGoerli: "https://opt-goerli.g.alchemy.com/v2/%s",
+	client.ChainIDs.Optimism:       "https://opt-mainnet.g.alchemy.com/v2/%s",
+	client.ChainIDs.ArbitrumGoerli: "https://arb-goerli.g.alchemy.com/v2/%s",
+	client.ChainIDs.Arbitrum:       "https://arb-mainnet.g.alchemy.com/v2/%s",
+	client.ChainIDs.PolygonMumbai:  "https://polygon-mumbai.g.alchemy.com/v2/%s",
+	client.ChainIDs.Polygon:        "https://polygon-mainnet.g.alchemy.com/v2/%s",
+}
+
+var localURLs = map[client.ChainID]string{
+	client.ChainIDs.Local: "http://localhost:8545",
+}
```

Comment:
Kind of nasty to have this lookup data here, but it's the easiest thing to do for now and still make it easy for the user to pass in the batteries-included type of connection information that is used by the Go Tableland client.